functions/__sf_section_jobs.fish

```diff
@@ -0,0 +1,44 @@
+# Jobs
+#
+
+function __sf_section_jobs -d "Show icon, if there's a working jobs in the background."
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_JOBS_SHOW true
+    __sf_util_set_default SPACEFISH_JOBS_PREFIX ""
+    __sf_util_set_default SPACEFISH_JOBS_SUFFIX " "
+    __sf_util_set_default SPACEFISH_JOBS_SYMBOL ✦
+    __sf_util_set_default SPACEFISH_JOBS_COLOR "blue"
+    __sf_util_set_default SPACEFISH_JOBS_AMOUNT_PREFIX ""
+    __sf_util_set_default SPACEFISH_JOBS_AMOUNT_SUFFIX ""
+    __sf_util_set_default SPACEFISH_JOBS_AMOUNT_THRESHOLD 1
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_JOBS_SHOW = false ]; and return
+
+    set jobs_amount (jobs | wc -l) # Zsh had a much more complicated command.
+
+    if test $jobs_amount -eq 0
+      return
+    end
+
+    if test $jobs_amount -le $SPACEFISH_JOBS_AMOUNT_THRESHOLD
+        set jobs_amount ''
+        set SPACEFISH_JOBS_AMOUNT_PREFIX ''
+        set SPACEFISH_JOBS_AMOUNT_SUFFIX ''
+    end
+
+    set SPACEFISH_JOBS_SECTION "$SPACEFISH_JOBS_SYMBOL$SPACEFISH_JOBS_AMOUNT_PREFIX$jobs_amount$SPACEFISH_JOBS_AMOUNT_SUFFIX"
+
+    __sf_lib_section \
```

Comment:
It looks like most uses of `__sf_lib_section` aren't indented after the line of the function call. I do prefer how it looks indented.

Would you mind indenting the other uses of `__sf_lib_section` as part of your PR?